git-merge.md

```diff
@@ -1,2 +1,27 @@
 12. Git Merge
 
+#git merge
```

Comment:
Try to use `# git merge` instead of `#git merge`.

The extra space makes the heading works.

like this:

# working heading

#not working heading